apps/reports/models.py

```diff
@@ -73,8 +75,11 @@ def get_context(self, request):
         ]
         external_reports = []
         for report in self.external_reports.raw_data:
-            report_data = report["value"]
+            report_data = report["value"].copy()
             report_data["external"] = True
+            report_data["last_updated"] = datetime.strptime(
+                report["value"]["last_updated"], "%Y-%m-%d"
```

Comment:
```suggestion
                report["value"]["last_updated"], "%b-%m-%d"
```
I think we want the abbreviated month instead of the full month. https://www.figma.com/proto/OZAfVKxh8fs8XE7RLnlbY7/PDPH-COVID19?node-id=1003%3A701&viewport=1007%2C-8587%2C0.4866814613342285&scaling=min-zoom. If so, I think we need to update internal links as well.

Reply:
This line just takes a string like "2020-01-01" and turns it into a `datetime.date` object; then, the date is sent to the template. The formatting itself is done in the template, so I'll change it there.

Reply:
I see. thanks